config/database.docker.yml

```diff
@@ -0,0 +1,19 @@
+default: &default
+  encoding: unicode
+  adapter:  postgresql
+  host:     postgres
+  pool:     20
+  username: <%= ENV['DATABASE_USER'] %>
+  password:
```

Comment:
Why empty?

Reply:
Coz by default postgres container does not set password